client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -183,43 +179,63 @@ def get_curl(self):
         self,
-        x,
+        x: List[Any],
         compress=False,
-        max_retries=5,
-        always_retry_404=True,
-        always_retry_429=True,
-    ):
+        max_retries: int = http_session.DEFAULT_MAX_RETRIES,
+        always_retry_404=False,
+        always_retry_429=False,
+        retry_status: Set[int] = http_session.DEFAULT_STATUS_FORCELIST,
+        backoff_factor: float = http_session.DEFAULT_BACKOFF_FACTOR,
+        prediction_id: str = None,
+        ) -> Dict[str, Any]:
         """
         Makes a prediction using input `x`.
 
-        .. versionadded:: 0.13.14
-           The `always_retry_404` parameter.
-        .. versionadded:: 0.13.12
-           The `always_retry_429` parameter.
+        .. deprecated:: 0.22.0
+           The ``always_retry_404`` parameter is deprecated. 404 is included by default in the
+           ``retry_status`` parameter.  Default is 13 retries over 10 minutes.  This behavior
+           can be changed by adjusting ``max_retries`` and ``backoff_factor``.
+        .. deprecated:: 0.22.0
+           The ``always_retry_429`` parameter is deprecated. 429 is included by default in the
+           ``retry_status`` parameter.  Default is 13 retries over 10 minutes.  This behavior
+           can be changed by adjusting ``max_retries`` and ``backoff_factor``.
+        .. versionadded:: 0.22.0
+           The ``retry_status`` parameter.
+        .. versionadded:: 0.22.0
+           The ``backoff_factor`` parameter.
+        .. versionadded:: 0.22.0
+           The ``prediction_id`` parameter.
 
         Parameters
         ----------
         x : list
             A batch of inputs for the model.
         compress : bool, default False
             Whether to compress the request body.
-        max_retries : int, default 5
-            Maximum number of times to retry a request on a connection failure.
-        always_retry_404 : bool, default True
-            Whether to retry on 404s indefinitely. This is to accommodate model deployment warm-up.
-        always_retry_429 : bool, default True
-            Whether to retry on 429s indefinitely. This is to accommodate third-party cluster
+        max_retries : int, default 13
+            Maximum number of retries on status codes listed in ``retry_status``.
+        always_retry_404 : bool, default False
+            Deprecated: Whether to retry on 404s indefinitely. This is to accommodate model deployment warm-up.
+        always_retry_429 : bool, default False
+            Deprecated: Whether to retry on 429s indefinitely. This is to accommodate third-party cluster
             autoscalers, which may take minutes to launch new pods for servicing requests.
-
+        retry_status : set, default {404, 429, 500, 503, 504}
+            Set of status codes, as integers, for which retry attempts should be made.  Overwrites default value.
+            Expand the set to include more: ```retry_status={404, 429, 500, 503, 504, <NEW_STATUS_CODE>}```
```

Comment:
Should we use just two backticks instead?

```suggestion
            Expand the set to include more: ``retry_status={404, 429, 500, 503, 504, <NEW_STATUS_CODE>}``
```

Three gets rendered as

![Screen Shot 2022-12-21 at 1 52 00 PM](https://user-images.githubusercontent.com/96442646/209010115-154e380f-7ed6-4df9-a516-0c21ea2d1dc0.png)

which I _think_ might not be the intention, but I'm not sure.